<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_main</class>
 <widget class="QMainWindow" name="frm_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_offene_leistungen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>1181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Offene Leistungen</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_offene_rechnungen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>1181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Offene Rechnungen</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableView" name="tbl_offene_leistungen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>1181</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tbl_offene_rechnungen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>1181</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_leistungen_erfassen">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>650</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Leistungen erfassen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_rechnung_erstellen">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>650</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Rechnung erstellen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_zahlung_erfassen">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>650</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Zahlung erfassen</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuStammdaten">
    <property name="title">
     <string>Stammdaten</string>
    </property>
    <addaction name="actionKunden_verwalten"/>
    <addaction name="actionAuftraggeber_verwalten"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuStammdaten"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionKunden_verwalten">
   <property name="text">
    <string>Kunden verwalten</string>
   </property>
  </action>
  <action name="actionAuftraggeber_verwalten">
   <property name="text">
    <string>Auftraggeber verwalten</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
